### PR TITLE
[eclipse/xtext-core#759] Use new EMF update site locations

### PR DESCRIPTION
**Pull request by kthoms**
Signed-off-by: Karsten Thoms <karsten.thoms@itemis.de>

### CODE REVIEW

**Review by cdietrich — APPROVED** [review]

**Review by cdietrich — CHANGES_REQUESTED** [review]
bad changes

File: composite/releases/compositeArtifacts.xml [hunk 1]
```diff
@@ -2,10 +2,10 @@
...
   <properties size='1'>
-    <property name='p2.timestamp' value='1425976704940'/>
+    <property name='p2.timestamp' value='1554465180693'/>
   </properties>
   <children size='4'>
-    <child location='http://download.eclipse.org/modeling/emft/mwe/updates/releases/'/>
+    <child location='http://download.eclipse.org/modeling/emf/emf/builds/release/'/>
     <child location='http://download.eclipse.org/modeling/tmf/xtext/updates/releases/'/>
     <child location='http://download.eclipse.org/modeling/tmf/xtext/updates/releases/xpand/'/>
     <child location='http://download.eclipse.org/modeling/emf/emf/updates/2.10.x/'/>
```
Comment (cdietrich) [inline composite/releases/compositeArtifacts.xml]: this one should be changed, not mwe

File: composite/releases/compositeContent.xml [hunk 2]
```diff
@@ -2,10 +2,10 @@
...
   <properties size='1'>
-    <property name='p2.timestamp' value='1425976704944'/>
+    <property name='p2.timestamp' value='1554465180693'/>
   </properties>
   <children size='4'>
-    <child location='http://download.eclipse.org/modeling/emft/mwe/updates/releases/'/>
+    <child location='http://download.eclipse.org/modeling/emf/emf/builds/release/'/>
     <child location='http://download.eclipse.org/modeling/tmf/xtext/updates/releases/'/>
     <child location='http://download.eclipse.org/modeling/tmf/xtext/updates/releases/xpand/'/>
     <child location='http://download.eclipse.org/modeling/emf/emf/updates/2.10.x/'/>
```
Comment (cdietrich) [inline composite/releases/compositeContent.xml]: this one should be changed. not mwe

File: composite/releases/compositeContent.xml [hunk 3]
```diff
@@ -2,10 +2,10 @@
 <?compositeMetadataRepository version='1.0.0'?>
 <repository name='Xtext All In One (Releases)' type='org.eclipse.equinox.internal.p2.metadata.repository.CompositeMetadataRepository' version='1.0.0'>
   <properties size='1'>
-    <property name='p2.timestamp' value='1425976704944'/>
+    <property name='p2.timestamp' value='1554465180693'/>
   </properties>
   <children size='4'>
-    <child location='http://download.eclipse.org/modeling/emft/mwe/updates/releases/'/>
+    <child location='http://download.eclipse.org/modeling/emf/emf/builds/release/'/>
```
Comment (cdietrich) [inline composite/releases/compositeContent.xml]: this is mwe,not emf

File: composite/releases/compositeArtifacts.xml [hunk 4]
```diff
@@ -2,10 +2,10 @@
 <?compositeArtifactRepository version='1.0.0'?>
 <repository name='Xtext All In One (Releases)' type='org.eclipse.equinox.internal.p2.artifact.repository.CompositeArtifactRepository' version='1.0.0'>
   <properties size='1'>
-    <property name='p2.timestamp' value='1425976704940'/>
+    <property name='p2.timestamp' value='1554465180693'/>
   </properties>
   <children size='4'>
-    <child location='http://download.eclipse.org/modeling/emft/mwe/updates/releases/'/>
+    <child location='http://download.eclipse.org/modeling/emf/emf/builds/release/'/>
```
Comment (cdietrich) [inline composite/releases/compositeArtifacts.xml]: this is mwe,not emf

File: composite/releases/compositeArtifacts.xml (hunk 1 above)
Comment (kthoms) [reply composite/releases/compositeArtifacts.xml]: Updated.

**Review by cdietrich — APPROVED** [review]

**Pull request merged**